app/src/main/java/org/mozilla/fenix/compose/SectionHeader.kt

```diff
@@ -22,10 +25,15 @@ fun SectionHeader(
     text: String,
     modifier: Modifier = Modifier
 ) {
+    val wallpaperState = components.appStore
+        .observeAsComposableState { state -> state.wallpaperState }.value
+
+    val wallpaperAdaptedTextColor = wallpaperState?.currentWallpaper?.textColor?.let { Color(it) }
```

Comment:
This could be a followup, but I think we should actually add a `color` parameter to `SectionHeader`, `HomeSectionHeaderContent`, and `PoweredByPocketHeader`. I don't think these components should need to observe the appstore.

Reply:
Agree, for when we would have all the home screen sections (including their list) in Compose. At the moment, hoisting the observation would imply that we need to observe in multiple places (all the different sections that use SectionHeader) and pass the color down. That would mean duplicating code in too many places.

Reply:
I think duplicating the code is fine given the fact that we currently lose out on previews of these composable.